src/client/theme-default/components/VPFlyout.vue

```diff
@@ -126,6 +120,7 @@ function onBlur() {
 }
 
 .text-icon {
+  flex: none;
```

Comment:
When space become scarce, Flexbox on the parent compresses this SVG icon. In one of my tests it ended up being 9px wide instead of the specified 14px wide.

Rule of thumb for SVG icons that are flex children: always use `flex: none` or `flex-shrink: 0`.